### PR TITLE
[content] v4.0 chrome extension article

### PR DESCRIPTION
**Pull request by markjmeier**
@robingustafsson We need to add links and images once available, but here is the first draft.

### CODE REVIEW

File: jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+layout: classic-docs
+title: Using the Load Impact Version 4.0 Chrome Extension
+description: "How do I use the Load Impact Chrome Extension to record user scenarios in version 4.0?"
+categories: [how-to-tutorials]
+order: 1
+---
+
+***
+
+<h1>Background</h1>
```
Comment (robingustafsson) [inline jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md]: You can use `# Background` for the same &lt;h1&gt; result here

File: jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The Load Impact Chrome Extension allows you to generate the bulk of your test scripts simply by browsing like a user would on your site or webapp.  The script created gives you a foundation which you can further edit, as required.
```
Comment (robingustafsson) [inline jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md]: I think "webapp" should be "web app"

File: jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md (hunk 1 above)
Comment (markjmeier) [reply jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md]: Yes, but then it shows up in the table of contents if you use:
```
- TOC
{:toc}
```
:D

**Comment by markjmeier** [comment]
Merged, Thanks @robingustafsson !

File: jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md (hunk 1 above)
Comment (robingustafsson) [reply jekyll/_v4/how-to-tutorials/load-impact-version-4-chrome-extension.md]: Ah, didn't know that 😄

**Pull request merged**